releases/latest/edge/bundle-airgap.yaml

```diff
@@ -0,0 +1,260 @@
+  katib-db:
+    charm: ./mysql-k8s_10afaca.charm
+    scale: 1
+    trust: true
+    constraints: mem=2G
+    resources:
+      mysql-image: 172.17.0.2:5000/canonical/charmed-mysql:753477ce39712221f008955b746fcf01a215785a215fe3de56f525380d14ad97
+  katib-db-manager:
+    charm: ./katib-db-manager_cb61fe0.charm
+    scale: 1
+    trust: true
+    resources:
+      oci-image: 172.17.0.2:5000/kubeflowkatib/katib-db-manager:v0.16.0-rc.1
+  katib-ui:
+    charm: ./katib-ui_d317886.charm
+    scale: 1
+    trust: true
+    resources:
+      oci-image: 172.17.0.2:5000/kubeflowkatib/katib-ui:v0.16.0-rc.1
+  kfp-api:
+    charm: ./kfp-api_5708923.charm
+    scale: 1
+    trust: true
+    resources:
+      oci-image: 172.17.0.2:5000/charmedkubeflow/api-server:2.0.0-alpha.7_20.04_1
+  kfp-db:
+    charm: ./mysql-k8s_10afaca.charm
+    scale: 1
+    trust: true
+    constraints: mem=2G
+    resources:
+      mysql-image: 172.17.0.2:5000/canonical/charmed-mysql:753477ce39712221f008955b746fcf01a215785a215fe3de56f525380d14ad97
+  kfp-persistence:
+    charm: ./kfp-persistence_a7d1ba7.charm
+    scale: 1
+    trust: true
+    resources:
+      oci-image: 172.17.0.2:5000/charmedkubeflow/persistenceagent:2.0.0-alpha.7_22.04_1
+  kfp-profile-controller:
+    charm: ./kfp-profile-controller_527ffbc.charm
+    scale: 1
+    trust: true
+    resources:
+      oci-image: 172.17.0.2:5000/python:3.7
+  kfp-schedwf:
+    charm: ./kfp-schedwf_31d7d73.charm
+    scale: 1
+    trust: true
+    resources:
+      oci-image: 172.17.0.2:5000/charmedkubeflow/scheduledworkflow:2.0.0-alpha.7_22.04_1
+  kfp-ui:
+    charm: ./kfp-ui_dd3a136.charm
+    scale: 1
+    trust: true
+    resources:
+      ml-pipeline-ui: 172.17.0.2:5000/ml-pipeline/frontend:2.0.0-alpha.7
+  kfp-viewer:
+    charm: ./kfp-viewer_17bb76d.charm
+    scale: 1
+    trust: true
+    resources:
+      kfp-viewer-image: 172.17.0.2:5000/charmedkubeflow/viewer-crd-controller:2.0.0-alpha.7_22.04_1
+  kfp-viz:
+    charm: ./kfp-viz_874d439.charm
+    scale: 1
+    trust: true
+    resources:
+      oci-image: 172.17.0.2:5000/ml-pipeline/visualization-server:2.0.0-alpha.7
+  knative-eventing:
+    charm: ./knative-eventing_d160a86.charm
+    scale: 1
+    trust: true
+    options:
```

Comment:
I see that here we didn't define the `version: 1.8.0` like we did in serving

@orfeas-k @NohaIhab is this needed?

Reply:
It's not needed since knative-eventing is [1.8.0 by default](https://github.com/canonical/knative-operators/blob/main/charms/knative-eventing/config.yaml#L6) (was never upgraded)